### PR TITLE
add tracker

### PR DESCRIPTION
**Pull request by sugoma11**

### CODE REVIEW

File: src/vision/stingray_object_detection/scripts/stingray_object_detection/yolo_detector.py [hunk 1]
```diff
@@ -200,36 +206,28 @@ def detector(self, img):
...
-                            conf.cpu().detach().numpy())
-                        object_msg.top_left_x = int(
-                            xyxy[0].cpu().detach().numpy())
-                        object_msg.top_left_y = int(
-                            xyxy[1].cpu().detach().numpy())
-                        object_msg.bottom_right_x = int(
-                            xyxy[2].cpu().detach().numpy())
-                        object_msg.bottom_right_y = int(
-                            xyxy[3].cpu().detach().numpy())
+                        object_msg.name = label + ' ' + str(int(id))
```
Comment (vladushked) [inline src/vision/stingray_object_detection/scripts/stingray_object_detection/yolo_detector.py:226]: вот здесь у тебя ошибка возникла?
добавь в конструктор self.device = torch.device("gpu" if torch.cuda.is_available() else "cpu")

и потом здесь проверку if self.device == 'cpu' (дальше твой код)
else: (удаленный код)

File: src/vision/stingray_object_detection/scripts/stingray_object_detection/yolo_detector.py [hunk 2]
```diff
@@ -7,12 +7,12 @@
 from stingray_object_detection_msgs.srv import SetEnableObjectDetection, SetEnableObjectDetectionResponse, SetEnableObjectDetectionRequest
 from stingray_resources.utils import load_config
 from stingray_object_detection.utils import get_objects_topic, get_debug_image_topic
+from stingray_object_detection.tracker import Sort
```
Comment (vladushked) [inline src/vision/stingray_object_detection/scripts/stingray_object_detection/yolo_detector.py:10]: Переименуй класс в Tracker для понятности

File: src/vision/stingray_object_detection/scripts/stingray_object_detection/yolo_detector.py (hunk 1 above)
Comment (vladushked) [reply src/vision/stingray_object_detection/scripts/stingray_object_detection/yolo_detector.py:226]: потому что с cuda надо обязательно делать .cpu().detach().numpy()

**Review by vladushked — CHANGES_REQUESTED** [review]

**Review by vladushked — APPROVED** [review]
